### ISSUE TITLE
Rss-ticker installation success page opens every time firefox starts and new windows

### ISSUE THREAD

**Issue by cfinke** [cfinke/RSS-Ticker#53]
<b>What steps will reproduce the problem?</b>
1. Open firefox either fresh or restored session.
2. Open a new window.
<b>3.</b>

<b>What is the expected output? What do you see instead?</b>
Just the start page. Instead I get the http://www.chrisfinke.com/firstrun/rss-ticker.php?v=3.2.1 page as well every time I start firefox or ope na new window.

<b>What version of the product are you using? On what operating system?</b>
Windows Vista, Firefox 3.5.10, Rss-ticker 3.2.1

<b>Please provide any additional information below.</b>
Problem doesn't cause any damage, just annoying.
Might be fixed with a simple uninstall and reinstall but thought I'd report it first.

---

Issue imported from Google Code
Author: [shadow1w2](http://code.google.com/u/shadow1w2/)
Published: 2010-07-13T22:35:04.000Z
Status: New
Link: http://code.google.com/p/rss-ticker/issues/detail?id=53

**Comment by cfinke** [comment]
Forgot to mention that the problem occurred only after the most recent update.

---

Comment imported from Google Code
Author: [shadow1w2](http://code.google.com/u/shadow1w2/)
Published: 2010-07-13T22:36:35.000Z
Link: http://code.google.com/p/rss-ticker/issues/detail?id=53#c1

**Comment by cfinke** [comment]
Duplicate of #49.
